t/print_errors.t

```diff
@@ -0,0 +1,71 @@
+use strict;
+use warnings;
+use lib 't/lib';
+use PkgConfig::Capture;
+use Test::More;
+use PkgConfig;
+
+note "DEFAULT_SEARCH_PATH = $_" for @PkgConfig::DEFAULT_SEARCH_PATH;
+
+# adjust to test with real pkg-config
+my @pkg_config = ( $^X, $INC{'PkgConfig.pm'} );
+#my @pkg_config = ( 'pkg-config' );
+
+#  assuming nobody will create a library named after an extinct
+#  crocodile from the Miocene Riversleigh fauna.  
```

Comment:
I am definitely now going to name my next library after an extinct crocodile.

But... this is because I love reading about extinct animals (not just dinosaurs), and am a bit strange.

Reply:
There are plenty to choose from.  Pallimnarchus might be a good one.